docs/getting_started.rst

```diff
@@ -0,0 +1,394 @@
+the library but they are not enough on their own to accomplish many analyses. To that end, it is
+often helpful to be able to handle missing data in your own functions or exploratory summaries.
+Both the sentinel values and the boolean mask array help make this possible, where the sentinel values
+are typically more useful when implementing compiled operations and the boolean mask array is easier to use
+in a higher level API like Xarray or Numpy.  Only advanced users would likely ever need to worry
+about compiling their own functions.  Using Xarray functions and the boolean mask is
+generally enough to accomplish most tasks, and this mask is often more efficient to operate
+on due to its high on-disk compression ratio.  These examples better demonstrate the distinction between
+the two as well as ways in which they are commonly used:
+
+.. ipython:: python
+
+    import sgkit as sg
+    ds = sg.simulate_genotype_call_dataset(n_variant=1, n_sample=4, n_ploidy=2, missing_pct=.3, seed=4)
+    ds.call_genotype
+
+    # Count alternate alleles while omitting partial calls
+    ##############
+    # Using Xarray
+    ##############
+    import xarray as xr
+    alt_allele_count = xr.where(
+        # Identify where there are any missing calls across chromosomes
+        ds.call_genotype_mask.any(dim='ploidy'),
+        -1, # Return -1 if any one call for a chromosome is missing
+        (ds.call_genotype > 0).sum(dim='ploidy') # Otherwise, sum non-ref calls
+    )
+    # Note that only the first two samples have meaningful counts since
+    # at least one call is missing for the last two samples
+    alt_allele_count.values
+
+    #############
+    # Using Numba
+    #############
+    import numba
+    import numpy as np
+
+    def alt_allele_count(gt):
+        out = np.full(gt.shape[:2], -1, dtype=np.int64)
+        for i, j in np.ndindex(*out.shape):
+            if np.all(gt[i, j] >= 0):
+                out[i, j] = np.sum(gt[i, j] > 0)
+        return out
+
+    # Jit-compiled functions are often simpler with a single array input, since
+    # conditional logic based on sentinel values is easier to program with this API
+    numba.njit(alt_allele_count)(ds.call_genotype.values)
+
+Again, this is not necessarily a level of detail most users should need to worry about. It is important
+to understand though when constructing more complex workflows that take full advantage of the many
+tools in the `PyData <https://pydata.org/>`_ ecosystem.
+
+
+Chaining operations
+~~~~~~~~~~~~~~~~~~~
+
+`Method chaining <https://tomaugspurger.github.io/method-chaining.html>`_ is a common practice with Python
+data tools that improves code readability and reduces the probability of introducing accidental namespace collisions.
+Sgkit functions are compatible with this idiom by default and this example shows to use it in conjunction with
+Xarray and Xandas operations in a single pipeline:
```

Comment:
Cool new project? :smile:

Reply:
Still have a "Xandas" typo here.

Reply:
🤦 ty.  I read @tomwhite's comment twice and still couldn't see the problem for some reason.  Xandas does sound cool though.

https://github.com/pystatgen/sgkit/pull/278/commits/dcb78dba17a0b78aa7cef5bf118253dadc0f881f